versions/q-/qca.json

```
@@ -1,7 +1,12 @@
 {
   "versions": [
     {
-      "git-tree": "41239863ffd0059d8688502f6d6284785c6c8633",
+      "git-tree": "f719790dc5863b3724d0fed5a7179cb1ca41c871",
+      "version": "2.3.4",
+      "port-version": 3
+    },
+    {
+      "git-tree": "7f79752be7b3504a67e28497fa93964af42a3f18",
```

```suggestion
      "git-tree": "41239863ffd0059d8688502f6d6284785c6c8633",
```

Please don't modify an existing version.

---

Thanks for pointing it out.
I have accidentally run `./vcpkg x-add-version --overwrite-version qca` once before updating the version in `vcpkg.json`.